dts/bindings/clock/nordic,nrf51-hfxo.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF high-frequency crystal oscillator (nRF51 series)
+
+compatible: "nordic,nrf51-hfxo"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
+    const: 32000000
+
+  startup-time-us:
+    type: int
+    default: 1400
+    description: Startup time in microseconds.
```

Comment:
We should document how to select this value. Similar for other added bindings

Reply:
Do you know how to select this value for the 51? I used https://github.com/nrfconnect/sdk-nrf/pull/22369/files which only mentions 52, 53 and 54

Reply:
No, maybe @masz-nordic  knows?